src/openforms/payments/contrib/worldline/models.py

```diff
@@ -0,0 +1,63 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from onlinepayments.sdk.client import IMerchantClient
+from onlinepayments.sdk.communicator_configuration import CommunicatorConfiguration
+from onlinepayments.sdk.factory import Factory
+from onlinepayments.sdk.merchant.merchant_client import IHostedCheckoutClient
+
+from .constants import WorldlineEndpoints
+
+
+class WorldlineMerchant(models.Model):
+    label = models.CharField(
+        _("Label"),
+        max_length=255,
+        help_text=_("Human readable label"),
+    )
+    pspid = models.CharField(
+        _("PSPID"),
+        max_length=255,
+        help_text=_("Worldline PSPID"),
+    )
+
+    api_key = models.CharField(
+        _("API Key"),
+        max_length=255,
+        help_text=_("API Key created for the specified PSPID"),
+    )
+    api_secret = models.CharField(
+        _("API Secret"),
+        max_length=255,
+        help_text=_("API Secret created for the specified PSPID"),
+    )
+
+    endpoint = models.URLField(
+        _("Preset endpoint"),
+        choices=WorldlineEndpoints.choices,
+        default=WorldlineEndpoints.test,
+        help_text=_("Select a common preset endpoint"),
+    )
+
+    def __str__(self):
+        return self.label
+
+    def get_merchant_client(self) -> IMerchantClient:
+        configuration = CommunicatorConfiguration(
+            api_endpoint=self.endpoint,
+            api_key_id=self.api_key,
+            secret_api_key=self.api_secret,
+            authorization_type="v1HMAC",
+            integrator="openforms",  #  TODO: is this the correct value?
+            connect_timeout=5000,
+            socket_timeout=10000,
```

Comment:
we have `DEFAULT_TIMEOUT_REQUESTS` in the settings which you can customize with an envvar, it'd probably be nice to use that instead of some hardcoded magic numbers.

It's applied here: https://github.com/open-formulieren/open-forms/blob/cfe21393ebd22e3cd30673ea2c1e6fcd8d2b0afe/src/openforms/setup.py#L134C47-L134C71

So perhaps if these are not required options, we can just rely on that implicitly?